components/SearchForm.js

```diff
@@ -113,6 +130,11 @@ class SearchForm extends React.Component {
             onFocus={onFocus}
             autoComplete={autoComplete}
           />
+          {this.props.value && (
+            <ClearFilterButton as="button" ml={2} p={1}>
```

Comment:
We need a label for accessibility.

```suggestion
            <ClearFilterButton aria-label={this.props.intl.formatMessage({ defaultMessage: 'Clear search' })}>
```

You'll need to add `injectIntl` on the component.